contrib/jupyter/setup.py

```diff
@@ -14,5 +14,5 @@
     url='http://github.com/google/orbit',
     license='BSD 2-Clause License',
     description='Orbit capture utilities',
-    install_requires=[],
+    install_requires=["protobuf"],
```

Comment:
Sorry for nitpicking: It might make sense to stick to a fixed, maybe the one we use on the C++ side.

This would be `install_requires=["protobuf==3.9.1"],`. The latest version would be  `install_requires=["protobuf==3.15.6"],`

Reply:
No worries, I'm glad you are pointing this out. Will fix!

Reply:
Thanks. If the CI agrees I think this will be good to go!